scripts/get-version.py

```diff
@@ -0,0 +1,60 @@
+"""
+Obtain versions of Quipucords & co installed in environment.
+
+This script is used by CI to set test run metadata.
+"""
+import argparse
+import subprocess
+
+from camayoc import api
+
+
+def get_backend_version():
+    api_client = api.Client()
+    server_status = api_client.get("status/")
+    server_version = server_status.json().get("server_version")
+    return server_version
+
+
+def get_frontend_version():
+    pass
```

Comment:
maybe fail this one while it's incomplete?

Reply:
You think `ValueError` will be OK?

Since this all is for internal consumption only, I tried to be lightweight.

Reply:
Normally I would `raise NotImplementedError` in placeholder functions during development to be explicit. That way it I or someone else accidentally uses it, we get a rude blocking error.

Reply:
Fair enough. Implemented `NotImplementedError` ( :stuck_out_tongue_closed_eyes:  ) in 0c1ba0398aa551b1ee4116bbee804039cb9967c1